lib/extensions/string_extension.rb

```diff
@@ -0,0 +1,9 @@
+module StringExtension
+  def true?
+    /(true|t|yes|y|1)$/i.match(to_s).present?
+  end
+
+  def false?
```

Comment:
While it makes sense for logical completeness, I've just realised `.false?` isn't being used anywhere except the tests, either...

I'd suggest it should come out as well–makes the tests simpler and keeps potential mutants at bay 😄

Reply:
Ok. I would normally leave it there for the logical sense as you say, but probably in this service will never be used, as looking at the decision trees we do things when something is checked, not when something is *not* checked. Or we cover these under the If (checked) else (not checked) statement. So will remove it 👍